stack/corepersistence/queryindex/src/main/java/org/apache/usergrid/persistence/index/impl/EsEntityIndexBatchImpl.java

```diff
@@ -228,58 +217,29 @@ public EntityIndexBatch deindex( final IndexScope indexScope, final CandidateRes
 
     @Override
     public void execute() {
-        execute( bulkRequest.setRefresh( refresh ) );
+//        indexBatchBuffer.flush();
+        Observable.from(promises)
```

Comment:
i need to resolve all the promises though, not sure just resolving the observable will work without explicitly calling get